client/hotrod-client/pom.xml

```diff
@@ -177,6 +177,17 @@
          <artifactId>jakarta.transaction-api</artifactId>
       </dependency>
 
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-api</artifactId>
+      </dependency>
```

Comment:
While for the server, adding a hard dependency on the OTel API is ok-ish (although I'd still prefer some kind of lightweight wrapper), it should be absolutely optional for the client.

Reply:
OK let's introduce a lightweight wrapper, to address the optionality and to reduce (remove) the overall dependency from the OpenTelemetry API

Reply:
done with last (forced) push! thanks for the suggestion